silverback/runner.py

```diff
@@ -77,15 +77,29 @@ def __init__(self, app: SilverBackApp, *args, **kwargs):
         logger.info(f"Using {self.__class__.__name__}: max_exceptions={self.max_exceptions}")
 
     async def _block_task(self, block_handler: AsyncTaskiqDecoratedTask):
-        async for block in async_wrap_iter(chain.blocks.poll_blocks()):
+        async for block in async_wrap_iter(
```

Comment:
id like to get this one to honor the kwarg as well but may require identifying block handlers